src/server/status.rs

```diff
@@ -278,15 +391,55 @@ pub fn print_status_all(extended: bool, debug: bool, json: bool)
 {
     let os = detect::current_os()?;
     let methods = os.get_available_methods()?.instantiate_all(&*os, true)?;
+    let mut local_names = HashSet::new();
     let mut statuses = Vec::new();
     for meth in methods.values() {
         statuses.extend(
             meth.all_instances()?
             .into_iter()
-            .map(|i| i.get_status())
+            .map(|i| {
+                local_names.insert(String::from(i.name()));
+                i.get_status()
+            })
         );
     }
-    if statuses.is_empty() {
+
+    let mut tasks = Vec::new();
+    let creds_dir = credentials::base_dir()?;
+    for item in fs::read_dir(&creds_dir)? {
+        if let Ok(filename) = item?.file_name().into_string() {
+            if let Some(name) = filename.strip_suffix(".json") {
+                if !local_names.contains(name) {
+                    tasks.push((
+                        String::from(name),
+                        task::spawn(future::timeout(
+                            Duration::from_secs(1),
+                            RemoteStatus::new(name.into())
+                        ))
+                    ));
+                }
+            }
+        }
+    }
+    let remote_statuses = task::block_on(async {
+        let mut rv: Vec<RemoteStatus> = Vec::new();
+        for (name, task) in tasks {
```

Comment:
This can be [paralellized](https://docs.rs/futures-util/0.3.16/futures_util/stream/trait.StreamExt.html#method.buffer_unordered) but we can keep for future improvements.